### PR TITLE
Remove getClaims/setClaims from all Entity classes

### PR DESCRIPTION
**Pull request by Benestar**
This depends on #556 and #539

### CODE REVIEW

**Comment by thiemowmde** [comment]
How does this depend on `Entity` being removed? This shouldn't matter.

**Comment by thiemowmde** [comment]
As already asked in #556, please makes this patch independent from the other so the removal of the `setClaims` methods can be reviewed and merged without being forced to remove the class at the same time. I'm still not sure if it's worth the trouble to remove it. As I already said many times: The fact that something exists is not a problem in itself. It's deprecated. The problem is when it's used, which can be avoided. Not using something is not blocked on removing it. Just don't use it.

**Comment by thiemowmde** [comment]
Rebased.

File: src/Entity/Item.php [hunk 1]
```diff
@@ -303,24 +302,6 @@ public function setStatements( StatementList $statements ) {
 	}
 
 	/**
-	 * @deprecated since 1.0, use getStatements()->toArray() instead.
-	 *
-	 * @return Statement[]
-	 */
-	public function getClaims() {
```
Comment (thiemowmde) [inline src/Entity/Item.php]: This is unrelated to both commits in this pull request. Is this breaking change really necessary?

File: src/Entity/Property.php [hunk 2]
```diff
@@ -254,22 +253,4 @@ public function setStatements( StatementList $statements ) {
 		$this->statements = $statements;
 	}
 
-	/**
-	 * @deprecated since 1.0, use getStatements()->toArray() instead.
-	 *
-	 * @return Statement[]
-	 */
-	public function getClaims() {
```
Comment (thiemowmde) [inline src/Entity/Property.php]: As above.

**Comment by thiemowmde** [comment]
+2 from my side. @Benestar, feel free to merge this if it's ok for you.

During rebase I removed the first commit (it's still in #556). I change this patch so it only touches the Item and Property classes, and removes both getClaims and setClaims. The later is also removed in #556, so whatever patch is merged first, the other needs to be rebased. I suggest to merge this first.

**Comment by JeroenDeDauw** [comment]
Needs rebase

**Comment by Benestar** [comment]
Rebased and merged per Thiemo's +2
